doc/packages/build-support.md

```diff
@@ -0,0 +1,92 @@
+# Build Support {#sec-build-support}
+
+## `pkgs.substitute` {#pkgs-substitute}
+
+`pkgs.substitute` is a wrapper around [the `substitute` function](#fun-substitute) provided by the
```

Comment:
I'd think this is an implementation detail. Ask: "Does the user of the function need to know it."

The important part is, if it is an alias / provides the same functionality. But ... (what is different) is the important thing

Reply:
This sentence was mostly used to help reference the reader to the additional documentation, in case they (like myself) are relatively new to nix and may not know about alternative approaches to solve the same problem. Shall I remove it?

Reply:
I 'd use one of the predefined classes to put this into an note block.

```suggestion
:::{.note}
`pkgs.substitute` is a wrapper around [the `substitute` function](#fun-substitute) provided by the
...
:::
```

https://github.com/NixOS/nixpkgs/blob/master/doc/README.md#link-reference-definitions